### ISSUE TITLE
Centering of Content on Mobile

### ISSUE THREAD

**Issue by jamesqquick** [Learn-Build-Teach/learnbuildteach.com#30]
Content isn't centered on smaller screens for content.

![image](https://user-images.githubusercontent.com/5391915/219387040-92c2c2af-82a8-41c2-8d73-ad34be83e084.png)

**Comment by Canfry** [comment]
The content describes here is for admin access. We can't see our changes while working on it. I'm focusing on mobile version for all pages (except the home page that is being redesigned) we see if will work for the admin page.

**Comment by jamesqquick** [comment]
@Canfry how did you get access to see that admin tab? That should be hidden behind an authenticated user

**Comment by Canfry** [comment]
Yes I know I just see it on the image of the issue. I see the logout button. I don't have access to it. What I say is that it's difficult to work on it as we don't have access to it.

**Comment by Canfry** [comment]
I've made it responsive for normal routes and added an hamburger menu in the navbar. But as it is redesigned i didn't create a pull request!!! Let me know if you want me to create a pull request or see it first!
